bin/v-list-apis

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# info: list available APIs
+# options: [FORMAT]
+#
+# example: v-list-apis json
+
+#----------------------------------------------------------#
+#                Variables & Functions                     #
+#----------------------------------------------------------#
+
+# Argument definition
+format="${1:-shell}"
+
+# Includes
+# shellcheck source=/etc/hestiacp/hestia.conf
+source /etc/hestiacp/hestia.conf
+# shellcheck source=/usr/local/hestia/func/main.sh
+source $HESTIA/func/main.sh
+# load config file
+source_conf "$HESTIA/conf/hestia.conf"
+
+# JSON list function
+json_list() {
+    echo '{'
+    local quote COMMANDS_ARR
+    for api in $(ls $HESTIA/data/api/); do
```

Comment:
Error in Shell check 
8221	               ^---------------------^ SC2045 (error): Iterating over ls output is fragile. Use globs.